src/integration/pages/homePage/canonicalTests.js

```diff
@@ -0,0 +1,40 @@
+import { runCoreCanonicalTests } from '../../common';
+import runCrossPlatformTests from './crossPlatformTests';
+
+export default service => {
+  runCrossPlatformTests(service);
+  runCoreCanonicalTests();
+  describe.only('Page content', () => {
+    const pageData = window.SIMORGH_DATA.pageData;
+    // console.log(pageData);
+    const numberOfItems = pageData.curations.length;
+    // console.log(`lenght of curations: ${numberOfItems}`);
+    const firstItemHeadline = document.querySelector(
+      '[data-testid="topic-promos"] > li  h3 a',
+    );
+    // console.log(`First headline: ${firstItemHeadline}`);
+
+    const h1 = document.querySelector('h1');
+    const hierarchicalGrid = document.querySelector(
+      '[data-testid="hierarchical-grid"]',
+    );
+    const numberOfcurations = document.querySelectorAll('h2').length;
+
+    it('H1 is displayed', () => {
+      expect(h1).toBeInTheDocument();
+    });
+
+    it('hierarchical grid is displayed', () => {
+      expect(hierarchicalGrid).toBeInTheDocument();
+      expect(numberOfcurations).toEqual(18);
```

Comment:
Yeah I was thinking that initially but I wasn't sure if that was the right thing to do since we are using fixture data; but I will change it to use that. So by get the number of curation you mean by utilising the `numberOfcurations`?